CODEOWNERS

```diff
@@ -2,19 +2,20 @@
 /addons @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tkg-addons-owners
 /addons/pinniped @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tkg-iam-owners
 /cmd/cli @vmware-tanzu/tanzu-framework-reviewers @Anuj2512
+/cmd/cli/plugin/package @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tkg-addons-owners
+/cmd/cli/plugin/pinniped-auth @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tkg-iam-owners
+/cmd/cli/plugin/secret @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tkg-addons-owners
 /cmd/cli/plugin/tkr @vmware-tanzu/tanzu-framework-reviewers @prkalle
 /pkg/v1/cli @vmware-tanzu/tanzu-framework-reviewers @miclettej @Anuj2512
 /pkg/v1/providers @vmware-tanzu/tanzu-framework-reviewers @vmware-tanzu/tanzu-framework-lcm-reviewers @Anuj2512
+/pkg/v1/providers/tests/unit @vmware-tanzu/tkg-infra-x
```

Comment:
Please add vmware-tanzu/tanzu-framework-reviewers to this as well as a fallback or PR will be be held up until someone from tkg-infra-x chimes in

Reply:
actually, agree with @zjs of just adding vmware-tanzu/tkg-infra-x to /providers.
There is also a related PR #1049 that is already  updating the same file. If it is alright, let me incorporate this change (with the sorting) into that instead.